packages/polymath-shared/babel.config.js

```diff
@@ -0,0 +1,12 @@
+const base = require('../../config/babel.config.js');
+module.exports = {
+  babelrcRoots: ['../packages/*'],
+  presets: ['@babel/flow', '@babel/env', '@babel/react'],
```

Comment:
`@babel/react` this stuff might fail as it's not in the package.json. But except if you plan to have some React components in this package, you could remove this.

Reply:
True that, we don't need this for now 👍